pkg/postage/postagecontract/contract.go

```diff
@@ -305,7 +310,69 @@ func (c *postageContract) TopUpBatch(ctx context.Context, batchID []byte, topUpA
 	return ErrBatchTopUp
 }
 
+func (c *postageContract) ExpireBatches(ctx context.Context) error {
```

Comment:
These functions are added in the postage contract. So they need to be tested here. This code is not being tested in the storage incentives agent tests as you use mock logic there.

This is not the first time that you have missed code coverage completely.

Reply:
i didnt miss this, the postage contract wasnt complete so creating a test didnt make any sense, also i cover minimum of 85% when writing tests :)